docs/02-for-app-authors/03-appdata-guidelines/index.md

```diff
@@ -242,11 +273,19 @@ It should look something like this:
 
 Applications must supply a releases tag in their appdata to pass
 validation. Please make sure to also follow the [quality guidelines](/docs/for-app-authors/appdata-guidelines/quality-guidelines#release-notes)
-while writing release notes. Releases in appdata should look like this:
+while writing release notes.
+
+Paragraphs in release description must not exceed 15 and they must not
+contain any direct URLs. Flathub supports showing a detailed release
+note link when `<url type="details">` tag is used. Release dates must
+not be in the future and versions must be [properly ordered](https://www.freedesktop.org/software/appstream/docs/chap-AppStream-Misc.html#spec-vercmp-recommendations).
+
+Releases in appdata should look like this:
 
 ```xml
 <releases>
-  <release version="X.Y.Z" date="YYYY-MM-DD">
+  <release version="1.0.1" date="YYYY-MM-DD">
```

Comment:
Kinda curious how you fill out version now, but don't date :)

Reply:
Thought I filled it already